Makefile.pre.in

```diff
@@ -1293,6 +1297,8 @@ PYTHON_HEADERS= \
 		$(srcdir)/Include/internal/pycore_uop_metadata.h \
 		$(srcdir)/Include/internal/pycore_warnings.h \
 		$(srcdir)/Include/internal/pycore_weakref.h \
+		$(srcdir)/Modules/_testcapi/parts.h \
+		$(srcdir)/Python/condvar.h \
```

Comment:
`condvar.h` is a dependency of `thread.c` and `ceval_gil.c`; the former seems to be handled correctly in the Makefile, but the latter is all wrong[^1]. This should be fixed. Perhaps `ceval_gil.o` should get its own explicit list of dependencies.

[^1]: moreover, there are two `Python/ceval.o` targets (!)